tools/NuGet/template-nuget/DynamoVisualProgramming.Core.nuspec

```diff
@@ -9,39 +9,80 @@
         <projectUrl>https://github.com/DynamoDS/Dynamo</projectUrl>
         <icon>content\images\logo.png</icon>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
-        <description>Assemblies required to start a DynamoModel and execute DesignScript code. Contains the following files:
-        1) DynamoApplications.dll
-        2) DynamoCore.dll
-        3) DynamoInstallDetective.dll
-        4) DynamoShapeManager.dll 
-        5) DynamoUtilities.dll
-        6) ProtoCore.dll
-        7) DesignScriptBuiltin.dll
-        8) VMDataBridge.dll
-        9) DSCPython.dll</description>
+        <description>
+            Assemblies required to start a DynamoModel and execute DesignScript code. Contains the following files:
+            1) DynamoApplications.dll
+            2) DynamoCore.dll
+            3) DynamoInstallDetective.dll
+            4) DynamoShapeManager.dll
+            5) DynamoUtilities.dll
+            6) ProtoCore.dll
+            7) DesignScriptBuiltin.dll
+            8) VMDataBridge.dll
+            9) DSCPython.dll
+            10) DynamoPackages.dll
+        </description>
         <summary>This package contains the core assemblies for Dynamo.</summary>
-        <copyright>Copyright Autodesk 2020</copyright>
+        <copyright>Copyright Autodesk 2023</copyright>
         <dependencies>
             <group targetFramework=".NETFramework4.8">
+                <!--TODO see if we can remove this prism ref -->
                 <dependency id="Prism" version="4.1"/>
                 <dependency id="NUnit" version="2.6.3"/>
                 <dependency id="DynamoVisualProgramming.ZeroTouchLibrary" version="$Version$"/>
                 <dependency id="Newtonsoft.Json" version="13.0.1"/>
                 <dependency id="System.Collections.Immutable" version="1.7.1"/>
             </group>
+            <!--TODO - why do we reference nunit from dynamo core? This is an odd partial collection of dependencies to mark.-->
+            <group targetFramework="net6.0">
+                <dependency id="NUnit" version="3.13.3"/>
```

Comment:
Why do we need NUnit in the Core package ?
We already have another Test package...

Reply:
Yep, I agree, I have it on my list to try building the samples repo after I remove this dep (it's just existing) - but I thought I might do it in a followup PR.